turbopack/crates/turbo-tasks-memory/src/memory_backend.rs

```diff
@@ -685,6 +686,13 @@ impl Backend for MemoryBackend {
         }
     }
 
+    fn try_get_function_id(&self, task_id: TaskId) -> Option<FunctionId> {
+        self.with_task(task_id, |task| match &task.ty {
+            TaskType::Persistent { ty } => ty.try_get_function_id(),
+            _ => None,
+        })
+    }
+
```

Comment:
This was the easiest way to thread through this information, but it is a temporary solution. I should be able to remove this API in a later PR:
- The manager should store the function id in the `CurrentTaskState` as a result of `dynamic_call` happening.
- Local tasks will re-use their parent task's task id, so an API like this will be misleading once that happens.
- The backend shouldn't need to know anything about local tasks, they should be handled entirely by the manager.